shared/api/network_forward.go

```diff
@@ -91,7 +91,7 @@ type NetworkForwardPut struct {
 	// Example: My public IP forward
 	Description string `json:"description" yaml:"description"`
 
-	// Forward configuration map (refer to doc/networks.md)
+	// Forward configuration map (refer to doc/network-forwards.md)
```

Comment:
Can you rebase that one into the commit which introduces the API?

Reply:
Yep that was an oversight, hence the short commit message.